modules/assim.batch/R/pda.load.data.R

```diff
@@ -38,13 +40,19 @@ load.pda.data <- function(settings, con) {
     
     vars.used.index <- which(format$vars$bety_name %in% c(inputs[[i]]$variable.name))
     
+    if(format$file_name == "AmeriFlux.level2.h.nc"){
+      time.row <- NULL
+    }else{
+      time.row <- which(format$vars$bety_name %in% c("year", "month", "week", "day", "hour", "minute", "second"))
+    }
```

Comment:
Rather than adding it to pda.load.data, can we move it into the retrieval of the format itself and remove it from calc.benchmark? That way we're consistent and don't have to replicate this in other places that use load.data